worlds/ffmq/Client.py

```diff
@@ -71,7 +71,7 @@ async def game_watcher(self, ctx):
         received = await snes_read(ctx, RECEIVED_DATA[0], RECEIVED_DATA[1])
         data = await snes_read(ctx, READ_DATA_START, READ_DATA_END - READ_DATA_START)
         check_2 = await snes_read(ctx, 0xF53749, 1)
-        if check_1 in (b'\x00', b'\x55') or check_2 in (b'\x00', b'\x55'):
+        if check_1 != b'01' or check_2 != b'01':
```

Comment:
I assume this was snuck in as a quick simplification? Doesn't change any behaviors or anything?

Reply:
It makes it less likely that hard resetting in bizhawk will erroneously send checks or game completion.